charts/vaultwarden/values.yaml

```diff
@@ -173,6 +179,111 @@ ingress:
   ##   - Support for multiple TLS hostnames.
   ##
 
+## @section Probe Parameters
+##
+
+## Liveness probe configuration
+##
+livenessProbe:
+  ## @param livenessProbe.enabled Enable liveness probe
+  ##
+  enabled: true
+  ## @param livenessProbe.initialDelaySeconds Delay before liveness probe is initiated
+  ##
+  initialDelaySeconds: 200
+  ## @param livenessProbe.timeoutSeconds How long to wait for the probe to succeed
+  ##
+  timeoutSeconds: 1
+  ## @param livenessProbe.periodSeconds How often to perform the probe
+  ##
+  periodSeconds: 10
+  ## @param livenessProbe.successThreshold Minimum consecutive successes for the probe to be considered successful
+  ##
+  successThreshold: 1
+  ## @param livenessProbe.failureThreshold Minimum consecutive failures for the probe to be considered failed
+  ##
+  failureThreshold: 10
+
+## Readiness probe configuration
+##
+readinessProbe:
+  ## @param readinessProbe.enabled Enable readiness probe
+  ##
+  enabled: true
+  ## @param readinessProbe.initialDelaySeconds Delay before readiness probe is initiated
+  ##
+  initialDelaySeconds: 5
+  ## @param readinessProbe.timeoutSeconds How long to wait for the probe to succeed
+  ##
+  timeoutSeconds: 1
+  ## @param readinessProbe.periodSeconds How often to perform the probe
+  ##
+  periodSeconds: 10
+  ## @param readinessProbe.successThreshold Minimum consecutive successes for the probe to be considered successful
+  ##
+  successThreshold: 1
+  ## @param readinessProbe.failureThreshold Minimum consecutive failures for the probe to be considered failed
+  ##
+  failureThreshold: 3
+
+
+## Startup probe configuration
+##
+startupProbe:
+  ## @param startupProbe.enabled Enable startup probe
+  ##
+  enabled: false
+  ## @param startupProbe.initialDelaySeconds Delay before startup probe is initiated
+  ##
+  initialDelaySeconds: 60
+  ## @param startupProbe.timeoutSeconds How long to wait for the probe to succeed
+  ##
+  timeoutSeconds: 1
+  ## @param startupProbe.periodSeconds How often to perform the probe
+  ##
+  periodSeconds: 10
+  ## @param startupProbe.successThreshold Minimum consecutive successes for the probe to be considered successful
+  ##
+  successThreshold: 1
+  ## @param startupProbe.failureThreshold Minimum consecutive failures for the probe to be considered failed
+  ##
+  failureThreshold: 10
+
+## @param customLivenessProbe Custom configuration for liveness probe
+##
+customLivenessProbe: null
```

Comment:
The idea was to provide even more configurability by allowing a custom probe to be used (instead of httpGet). This is quite difficult to offer with the existing (non custom) probe structure. I guess that not many people would change the probe to something different, so I am removing the custom option.